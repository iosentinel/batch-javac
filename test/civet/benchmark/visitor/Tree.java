package benchmark.visitor;

public abstract class Tree {
	abstract int accept(TreeVisitor v);
}
