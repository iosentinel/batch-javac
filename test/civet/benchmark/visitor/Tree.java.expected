package benchmark.visitor;

public abstract class Tree {
    
    public Tree() {
        super();
    }
    
    abstract int accept(TreeVisitor v);
    
    abstract int accept$1002();
    
    abstract int accept$1007();
    
    abstract int accept$1012();
}